EduID/Flows/CreateEduID/ViewModels/CreatePincodeAndBiometricAccessViewModel.swift

```diff
@@ -89,8 +89,10 @@ final class CreatePincodeAndBiometricAccessViewModel: NSObject {
                         self.createIdentity(for: object as? EnrollmentChallenge, completion: completion)
                     }
                 } catch {
+                    let _ = try await TiqrControllerAPI.sendDeactivationPhoneCodeForSpWithRequestBuilder()
```

Comment:
This can be simplified to:
```
let result = try await TiqrControllerAPI.sendDeactivationPhoneCodeForSp()
```

Reply:
I tried didn't work very well! also I decided to omit the result by let _ to avoid Xcode editor's warnings.